src/prasadSDec2019/Assignment3/Question1.java

```diff
@@ -0,0 +1,25 @@
+package prasadSDec2019.Assignment3;
+
+import java.util.Scanner;
+
+public class Question1 {
+	void positiveNegitive() {
+		Scanner scn=new Scanner(System.in);
+		System.out.println("Enter the number of array");
+		int num=scn.nextInt();
+		System.out.println("Elements present in array are");
+		int a[]=new int[num];
+		for(int i=0;i<a.length;i++) {
+			a[i]=scn.nextInt();
```

Comment:
where is logic @PrasadSDec2019 
i can't see that you are separating things, i m closing this PR do all assignment proper and then only raise PR

Reply:
Complete this code in next PR